websites/L/Library Of Babel/metadata.json

```diff
@@ -0,0 +1,22 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.10",
+	"author": {
+		"id": "358402930191106049",
+		"name": "ion606"
+	},
+	"service": "Library Of Babel",
+	"description": {
+		"en": "A rich presence for libraryofbabel.info"
+	},
+	"url": "libraryofbabel.info",
+	"version": "1.0.0",
+	"logo": "https://libraryofbabel.info/img/hex802.png",
+	"thumbnail": "https://libraryofbabel.info/img/hex802.png",
```

Comment:
> The thumbnail should preferably be a [wide promotional card](https://i.imgur.com/3QfIc5v.jpg) or a [screenshot](https://i.imgur.com/OAcBmwW.png) if the first is not available.

Reply:
ohhhh I see, would something like this be better? 
![image](https://github.com/PreMiD/Presences/assets/58801387/62caa878-2137-4466-a4d8-400868981ccc)